website/docs/r/netapp_volume.html.markdown

```diff
@@ -0,0 +1,129 @@
+  resource_group_name = "${azurerm_resource_group.example.name}"
+}
+
+resource "azurerm_netapp_pool" "example" {
+  name                = "example-netapppool"
+  location            = "${azurerm_resource_group.example.location}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  account_name        = "${azurerm_netapp_account.example.name}"
+  service_level       = "Premium"
+  size_in_4_tb        = 1
+}
+
+resource "azurerm_netapp_volume" "example" {
+  name                = "example-netappvolume"
+  location            = "${azurerm_resource_group.example.location}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  account_name        = "${azurerm_netapp_account.example.name}"
+  pool_name           = "${azurerm_netapp_pool.example.name}"
+  creation_token      = "my-unique-file-path"
+  service_level       = "Premium"
+  subnet_id           = "${azurerm_subnet.example.id}"
+  usage_threshold     = 100
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the NetApp Volume. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group where the NetApp Volume should be created. Changing this forces a new resource to be created.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+* `account_name` - (Required) The name of the NetApp account in which the NetApp Pool should be created. Changing this forces a new resource to be created.
+
+* `creation_token` - (Required) A unique file path for the volume. Used when creating mount targets. Changing this forces a new resource to be created.
+
+* `pool_name` - (Required) The name of the NetApp pool in which the NetApp Volume should be created. Changing this forces a new resource to be created.
+
+* `service_level` - (Required) The service level of the file system. Valid values include `Premium`, `Standard`, or `Ultra`.
+
+* `subnet_id` - (Required) The ID of a Subnet in which the NetApp Volume, which must have the delegation Microsoft.NetApp/volumes. Changing this forces a new resource to be created.
+
+* `usage_threshold` - (Required) The maximum Storage Quota in Gigabytes allowed for a file system.
+
+* `export_policy_rule` - (Optional) One or more `export_policy_rule` block defined below.
+
+---
+
+An `export_policy_rule` block supports the following:
+
+* `rule_index` - (Required) The index number for the rule.
+
+* `allowed_clients` - (Required) Client ingress specification as list with IPv4 CIDRs, IPv4 host addresses.
+
+* `cifs` - (Required) Allows CIFS protocol.
+
+* `nfsv3` - (Required) Allows NFSv3 protocol.
+
+* `nfsv4` - (Required) Allows NFSv4 protocol.
+
+* `unix_read_only` - (Required) Read only file system type on unix.
+
+* `unix_read_write` - (Required) Read and write file system type on unix.
```

Comment:
could we improve the wording here:
```
* `rule_index` - (Required) The index number of the rule.


* `cifs` - (Required) Is the CIFS protocol allowed?

* `unix_read_only` - (Required) Is the file system on unix read only?
```

and so forth

Reply:
fixed